prime-router/build.gradle.kts

```diff
@@ -684,7 +684,7 @@ dependencies {
     implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.13.2") {
         exclude(group = "org.yaml", module = "snakeyaml")
     }
-    implementation("com.fasterxml.jackson.core:jackson-databind:2.13.1")
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.13.2.2")
```

Comment:
I don't know why this appears here, I checked master and it seems that master is using 2.13.2.2